### ISSUE TITLE
Can't use Filters button with username inside Kanban

### ISSUE THREAD

**Issue by Zen3515** [go-vikunja/api#84]
### Description

When inside Kanban view, and trying to use Filters button with user filter, the backend error, all the tasks become empty.

From the dev tool I saw response status 500  
From the backend docker logs I saw this
```bash
2023-10-03T14:10:33.47381376Z: ERROR	▶ handler/ReadAllWeb 198b Error 1054 (42S22): Unknown column 'username' in 'where clause'
```

### More context
- With other views like table view, it works fine.
- I'm using the latest unstable, this problem has been here since day one that I use vikunja (probably about 1.5 years ago)
- I use docker
- I use OIDC from Authelia
- I can reproduce this with new setup using similar config (OIDC, docker)

### Vikunja Frontend Version

unstable (I think it's irrelevant since problem exist at least 1.5 years)

### Vikunja API Version

unstable (I think it's irrelevant since problem exist at least 1.5 years)

### Can you reproduce the bug on the Vikunja demo site?

Yes

### Screenshots

![image](https://github.com/go-vikunja/api/assets/7106408/6b71df46-af13-45b2-b6af-84965a967cc5)
![image](https://github.com/go-vikunja/api/assets/7106408/ce89aa89-d626-41a4-ae0c-3e45a7983fff)
![image](https://github.com/go-vikunja/api/assets/7106408/1702121e-9eb5-4611-a488-8c8c79c49a3a)

**Comment by kolaente** [comment]
This is definitely a bug. I'll take a look.

**Comment by letshin** [comment]
@kolaente Yes, this is the same as https://community.vikunja.io/t/filters-embed-into-url-and-a-related-bug-on-kanban-view/1661/2
